qa/tasks/cephfs/test_forward_scrub.py

```diff
@@ -46,6 +47,12 @@ def _get_paths_to_ino(self):
 
         return inos
 
+    def _check_for_presence_of_MDS_damage(self):
+        return "MDS_DAMAGE" in self.mds_cluster.mon_manager.get_mon_health()['checks']
+
+    def _check_for_absence_of_MDS_damage(self):
+        return "MDS_DAMAGE" not in self.mds_cluster.mon_manager.get_mon_health()['checks']
```

Comment:
do we really need two functions for doing almost the same thing? how about having just method named `_is_mds_damage()` (or any other similar name) and negate it while using when you want no mds damage

Reply:
Updated, PTAL